src/modelinstance.cpp

```diff
@@ -542,6 +542,11 @@ Status ModelInstance::reloadModel(const ModelConfig& config, const DynamicModelP
             getName(), getVersion(), predictRequestsHandlesCount);
         std::this_thread::sleep_for(std::chrono::milliseconds(UNLOAD_AVAILABILITY_CHECKING_INTERVAL_MILLISECONDS));
     }
+    if ((this->config.isCustomLoaderRequiredToLoadModel()) && (isCustomLoaderConfigChanged)) {
+        // unloading and the loading back the model
+        isCustomLoaderConfigChanged = false;
+        unloadModel(isCustomLoaderConfigChanged);
```

Comment:
Minor - rename isCustomLoaderConfigChanged to isPermanentStatus = false?

This will make this more self-explanatory